### PR TITLE
Game Class

### PR DESCRIPTION
**Pull request by kernohad**

### CODE REVIEW

File: src/Model/GamePackage/Game.java [hunk 1]
```diff
@@ -204,7 +204,7 @@ public void setCurrentPlayer(Player currentPlayer) {
     /**********************************************************************
      * Rolls both die
      *********************************************************************/
-    public void rollDie() {
+    public void rollDies() {
```
Comment (Dcbaas) [inline src/Model/GamePackage/Game.java]: Needs to be "Dice." Dies isn't a word.

File: src/Model/GamePackage/Game.java [hunk 2]
```diff
@@ -310,52 +307,69 @@ public void movePlayer(Player player, int dieVal1, int dieVal2) {
...
+     * wallet.
+     *
+     * @param propertySquare the property that the mortgage is being lifted.
+     *********************************************************************/
+    public void payMortgage(PropertySquare propertySquare){
+       propertySquare.setMortgaged(false);
+       propertySquare.getOwner().pay((int)Math.ceil(propertySquare.getMORTGAGE_VAL() * 1.1));
+     }
 
-    public void auctionProperty() {
```
Comment (Dcbaas) [inline src/Model/GamePackage/Game.java]: If this functionality was removed here, is it implemented somewhere else?

File: src/Model/GamePackage/Game.java [hunk 3]
```diff
@@ -310,52 +307,69 @@ public void movePlayer(Player player, int dieVal1, int dieVal2) {
...
+                    chanceDeck.drawCard():comunityChestDeck.drawCard());
     }
 
 
     /**********************************************************************
      * This method checks for the list of actions that the card has and
      * performs the necessary actions.
      *
-     * @param cardToUse
+     * @param cardToUse the card that is going to be used
```
Comment (Dcbaas) [inline src/Model/GamePackage/Game.java]: What is the return condition?

File: src/Model/GamePackage/Game.java [hunk 4]
```diff
@@ -483,26 +502,23 @@ public void cardMoveNearest(int typeID){
         //Moves the player to the nearest specific location.
         board.setPlayerPosition(currentPlayer, closestSquare.getPOSITION());
         currentPlayer.setPosition(closestSquare.getPOSITION());
-
-        //TODO: NOTE:is there more to this method ? do we have to check if the player landed on a typle owned by someone else?
     }
 
     /**********************************************************************
      * Sets a player inJail status to -1, so the player can scape jail
```
Comment (Dcbaas) [inline src/Model/GamePackage/Game.java]: "so the player can leave jail

File: src/Model/GamePackage/Game.java (hunk 2 above)
Comment (kernohad) [reply src/Model/GamePackage/Game.java]: We decided this was going to be implemented by the GUI/controller. This method will just make calls to many other Game methods based on use input. Same with Trade.

**Comment by kernohad** [comment]
I fixed the miner changes mentioned above. Auction and trade will not be implemented here. Those will be handled by the controler/GUI based on user input. Not exactly sure the design for those right now. But we know they will not be in Game. Also, those are features we can cut if need for release one.

**Pull request merged**